src/style.css

```diff
@@ -0,0 +1,77 @@
+body {
+  background-color: #f6f6f6;
+}
+
+.to-do-list {
+  width: 90%;
+  height: 97vh;
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.to-do-list .list {
+  background-color: white;
+  padding: 0;
+  width: 60%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  overflow-x: hidden;
+  box-shadow: 0 0 6px 1px #00000045;
+}
```

Comment:
- [ ] The project has some missing icons on the right side of the task. I am attaching a screenshot below to compare with the given design.


### Your design 


![Screenshot from 2021-09-21 23-25-58](https://user-images.githubusercontent.com/22434660/134223454-72298cd4-c881-4177-8d9e-686d08739058.png)

### Given design
![todo](https://user-images.githubusercontent.com/22434660/134223466-c2f39f65-44e0-4b35-aa8e-2e3b1ffb1064.png)